backend/satellite_tools/reposync.py

```diff
@@ -360,14 +360,32 @@ def clear_ssl_cache():
     shutil.rmtree(ssldir, True)
 
 
+def verify_certificates_dates(certs):
+    cert = ""
+    isCert = False
+    valid = True
```

Comment:
I would suggest this change together with other one to make this validation not passing when the string does not even contain a valid certificate string at all.

```suggestion
    valid = None
```